common/dna_data/wc_orc_dna.txt

```diff
@@ -0,0 +1,825 @@
+ 		gene_bs_mouth_lower_lip_pad={ "mouth_lower_lip_pad_pos" 255 "mouth_lower_lip_pad_pos" 0 }
+ 		gene_bs_mouth_lower_lip_width={ "mouth_lower_lip_width_pos" 255 "mouth_lower_lip_width_neg" 2 }
+ 		gene_bs_mouth_philtrum_def={ "mouth_philtrum_def_pos" 25 "mouth_philtrum_def_pos" 25 }
+ 		gene_bs_mouth_philtrum_shape={ "mouth_philtrum_shape_pos" 11 "mouth_philtrum_shape_pos" 11 }
+ 		gene_bs_mouth_philtrum_width={ "mouth_philtrum_width_pos" 133 "mouth_philtrum_width_pos" 41 }
+ 		gene_bs_mouth_upper_lip_def={ "mouth_upper_lip_def_pos" 13 "mouth_upper_lip_def_pos" 13 }
+ 		gene_bs_mouth_upper_lip_full={ "mouth_upper_lip_full_neg" 149 "mouth_upper_lip_full_pos" 5 }
+ 		gene_bs_mouth_upper_lip_profile={ "mouth_upper_lip_profile_neg" 33 "mouth_upper_lip_profile_pos" 42 }
+ 		gene_bs_mouth_upper_lip_width={ "mouth_upper_lip_width_pos" 255 "mouth_upper_lip_width_neg" 102 }
+ 		gene_bs_nose_forward={ "nose_forward_pos" 27 "nose_forward_pos" 30 }
+ 		gene_bs_nose_height={ "nose_height_pos" 255 "nose_height_pos" 68 }
+ 		gene_bs_nose_length={ "nose_length_neg" 63 "nose_length_pos" 2 }
+ 		gene_bs_nose_nostril_height={ "nose_nostril_height_pos" 161 "nose_nostril_height_pos" 216 }
+ 		gene_bs_nose_nostril_width={ "nose_nostril_width_neg" 13 "nose_nostril_width_neg" 87 }
+ 		gene_bs_nose_profile={ "nose_profile_pos" 113 "nose_profile_hawk_pos" 110 }
+ 		gene_bs_nose_ridge_angle={ "nose_ridge_angle_neg" 133 "nose_ridge_angle_pos" 22 }
+ 		gene_bs_nose_ridge_width={ "nose_ridge_width_pos" 255 "nose_ridge_width_pos" 165 }
+ 		gene_bs_nose_size={ "nose_size_neg" 179 "nose_size_neg" 11 }
+ 		gene_bs_nose_tip_angle={ "nose_tip_angle_pos" 59 "nose_tip_angle_pos" 176 }
+ 		gene_bs_nose_tip_forward={ "nose_tip_forward_pos" 55 "nose_tip_forward_neg" 177 }
+ 		gene_bs_nose_tip_width={ "nose_tip_width_neg" 79 "nose_tip_width_neg" 122 }
+ 		face_detail_cheek_def={ "cheek_def_02" 82 "cheek_def_01" 147 }
+ 		face_detail_cheek_fat={ "cheek_fat_02_pos" 200 "cheek_fat_04_pos" 238 }
+ 		face_detail_chin_cleft={ "chin_cleft" 14 "chin_cleft" 14 }
+ 		face_detail_chin_def={ "chin_def" 32 "chin_def" 94 }
+ 		face_detail_eye_lower_lid_def={ "eye_lower_lid_def" 255 "eye_lower_lid_def" 252 }
+ 		face_detail_eye_socket={ "eye_socket_01" 160 "eye_socket_02" 65 }
+ 		face_detail_nasolabial={ "nasolabial_01" 237 "nasolabial_01" 229 }
+ 		face_detail_nose_ridge_def={ "nose_ridge_def_neg" 0 "nose_ridge_def_neg" 12 }
+ 		face_detail_nose_tip_def={ "nose_tip_def" 0 "nose_tip_def" 23 }
+ 		face_detail_temple_def={ "temple_def" 0 "temple_def" 229 }
+ 		expression_brow_wrinkles={ "brow_wrinkles_04" 255 "brow_wrinkles_03" 202 }
+ 		expression_eye_wrinkles={ "eye_wrinkles_03" 193 "eye_wrinkles_01" 193 }
+ 		expression_forehead_wrinkles={ "forehead_wrinkles_03" 255 "forehead_wrinkles_01" 255 }
+ 		expression_other={ "cheek_wrinkles_both_01" 0 "cheek_wrinkles_both_01" 209 }
+ 		complexion={ "complexion_7" 185 "complexion_2" 88 }
+ 		gene_height={ "normal_height" 255 "normal_height" 141 }
```

Comment:
Thrall is a giant.
Also, I think his skin is too dark.
Also, I would adjust his chin to be more orcish.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/109343333-a25eba80-7886-11eb-81dc-db583864c845.png)
![image](https://user-images.githubusercontent.com/46384992/109343437-ce7a3b80-7886-11eb-84bf-2b3ac6f2d73a.png)

</details>

Reply:
Fixed in fd9a9d95a8b6b2b1a8093df114ff40525820457e.